pkg/collector/iis/iis.go

```diff
@@ -1514,6 +1531,15 @@ func (c *collector) collectW3SVC_W3WP(ctx *types.ScrapeContext, ch chan<- promet
 			continue
 		}
 
+		ch <- prometheus.MustNewConstMetric(
```

Comment:
Is this section necessary? We appear to be adding to the channel twice for the same metric here.

Reply:
Seems like there was an mistake/leftover from copy/paste